vizro-core/pyproject.toml

```diff
@@ -76,10 +76,10 @@ filterwarnings = [
   "ignore:(?s).*Pyarrow will become a required dependency of pandas:DeprecationWarning",
   # Ignore until plotly fixes so the warning is no longer raised:
   "ignore:When grouping with a length-1 list-like, you will need to pass a length-1 tuple to get_group:FutureWarning",
-  # Ignore warning when providing a custom format string to the KPI cards:
-  "ignore:Custom format string detected.",
```

Comment:
Do we not want this anymore?

Reply:
We never actually put the warning in - it's just mentioned in the docs. So this ignore was never actually needed, we just left it in accidentallly.

Reply:
Am I going crazy? I definitely added it in 😄 It's also in the docstring description of the KPI card functions, but I can't see the code somehow. Ahh, did we decide to remove it and instead just add "Warning: BLABLA" inside the docstrings?

If yes, could you additionally remove this from all KPI card functions then?

```
    Raises:
        UserWarning: If `value_format` or `reference_format` is provided, a warning is raised to make aware
            that only trusted user input should be provided.
```

And I am trying to trace back what we decided on, because I once definitely added it in 😆

Reply:
I think we did have that UserWarning once but then decided to not raise it and instead have this in the docstrings?

```
    **Warning:** Note that the format string provided to `value_format` and `reference_format` is being evaluated,
    so ensure that only trusted user input is provided to prevent
    [potential security risks](https://stackoverflow.com/questions/76783239/is-it-safe-to-use-python-str-format-method-with-user-submitted-templates-in-serv).
```

In which case, you can remove the `Raises` section in the KPI card functions as well.

Reply:
Thanks for checking! Done in [ff7a37b](https://github.com/mckinsey/vizro/pull/813/commits/ff7a37b2d1c75b79bf2a4727bff862d36a9b843f).